src/apiCalls.js

```diff
@@ -1,7 +1,15 @@
-// Your fetch requests will live here!
-// Write fetch call
-// Export function to scripts.js
-// Import function in scripts.js
-// Invoke function in scripts.js
+// Fetch requests
 
-console.log('I will be a fetch request!')
\ No newline at end of file
+const usersApi = 
```

Comment:
Awesome, it's great we had some direction on how to implement the API consumption and use it in our scripts.js file.